src/main/java/com/greghaskins/spectrum/Configuration.java

```diff
@@ -1,5 +1,9 @@
 package com.greghaskins.spectrum;
 
+/**
+ * Allows the injection of suite configuration during test definition. A wrapper for the
+ * suite which exposes configurables.
+ */
```

Comment:
Checkstyle forced me to write Javadoc as items stopped being internal when they switched packages.